assets/components/priceCta/priceCta.scss

```diff
@@ -85,15 +86,3 @@
   width: 100%;
   box-sizing: border-box;
 }
```

Comment:
If `component-price-cta__price` and `component-price-cta__cancel` are now gone from the component, can some of the corresponding styles go too?